.jenkins/validate_tutorials_built.py

```diff
@@ -5,13 +5,15 @@
 
 REPO_ROOT = Path(__file__).parent.parent
 
-# files that are ok to have 0 min 0 sec time, probably because they don't have any python code
-OK_TO_NOT_RUN = [
-    "beginner/basics/intro.html",  # no code
-]
+# files not ending in "tutorial.py" are not run by sphinx (see sphinx_gallery_conf in conf.py),
```

Comment:
Does this mean we don't actually have any tutorials we currently know to be broken?

Reply:
yes, or at least i am not aware of them